packages/core-ethereum/src/channel/index.ts

```diff
@@ -126,8 +139,8 @@ class ChannelFactory {
               this.coreConnector.hoprChannels.options.address,
               amount.toBN().toString(),
               this.coreConnector.web3.eth.abi.encodeParameters(
-                ['address', 'address'],
-                [(await account.address).toHex(), counterparty.toHex()]
+                ['bool', 'address', 'address'],
+                [false, (await account.address).toHex(), counterparty.toHex()]
```

Comment:
we can now use the ERC777 hook to fund & open a channel in one go, will be implemented in a different PR